content/design-systems-for-developers/react/en/distribute.md

```diff
@@ -207,12 +210,13 @@ git reset HEAD^
 Then we’ll update the changelog and commit it:
 
 ```
-# v0.1.0 (Tue Sep 03 2019)
+# v0.1.0 (Tue Mar 09 2021)
 
 - Created first version of the design system, with `Avatar`, `Badge`, `Button`, `Icon` and `Link` components.
 
 #### Authors: 1
-- Tom Coleman ([@tmeasday](https://github.com/tmeasday))
+
+- [your-usermane](https://github.com/your-username)
```

Comment:
```suggestion
- [your-username](https://github.com/your-username)
```

Reply:
Fixed with the latest commit. Marking this as solved